_posts/newsletters/2023-01-24-newsletter.md

```diff
@@ -0,0 +1,217 @@
+---
+layout: post
+title: US-RSE January 2023 Newsletter
+subtitle: January 2023 
+category: newsletter
+tags: [newsletter]
+---
+
+In this monthly newsletter, we share recent, current, and planned activities of the US-RSE Association, and related news that we think is of interest to US-RSE members. Newsletters are also available on our [website](https://us-rse.org/newsletters/) alongside the growing resources and information on the US-RSE Association. To receive our newsletter, [join US-RSE](https://us-rse.org/join/).
+
+In this issue:
+
+* [Steering Committee Election Results](#elections)
+* [First US-RSE Conference in 2023!](#conference2023)
+* [Steering Committee Recap](#sc-update)
+* [Community News (Funders Talk Series & DEI-WG Speaker Series)](#news)
+* [Interesting Events and Opportunities](#events)
+* [Interesting Reads, Videos or Podcasts](#reads)
+   * [Developer Stories Podcast](#developer_stories_podcast)
+   * [Code For Thought Podcast](#code_for_thought_podcast)
+   * [Sustain Podcast](#sustain_podcast)
+* [US-RSE Outreach](#outreach)
+* [US-RSE Financials](#financials)
+* [Get Involved](#involved)
+* [Recent Job Postings](#jobs)
+
+    
+<span style="color: black;">
```

Comment:
This is a surprise, do we need to adjust the site's CSS somewhere? The previous newsletter didn't seem to need it.

Reply:
Good catch. I agree. I don't think it's necessary. Is it an artifact of hackmd or random some copy/paste?

Reply:
@sandragesing - do we need this?

Reply:
I removed it below. I think we're good without it.

Reply:
I think it's something that was set intentionally in hackmd because Sandra has used colors at some point to indicate old vs new text ,so this was probably leftover from that